x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/vector/VectorSimilarityFunctionsIT.java

```diff
@@ -37,6 +37,7 @@ public static Iterable<Object[]> parameters() throws Exception {
         List<Object[]> params = new ArrayList<>();
 
         params.add(new Object[] { "v_cosine", VectorSimilarityFunction.COSINE });
+        params.add(new Object[] { "v_dot_product", VectorSimilarityFunction.DOT_PRODUCT });
```

Comment:
Now that I think about this - we will need to guard adding params with the corresponding capability so this doesn't error out on release builds